esrally/reporter.py

```diff
@@ -87,6 +87,33 @@ def format_as_csv(headers, data):
         return out.getvalue()
 
 
+def disk_usage_fields(stats):
+    return {
+        "inverted index": stats.disk_usage_inverted_index,
+        "stored fields": stats.disk_usage_stored_fields,
+        "doc values": stats.disk_usage_doc_values,
+        "points": stats.disk_usage_points,
+        "norms": stats.disk_usage_norms,
+        "term vectors": stats.disk_usage_term_vectors,
+        "total": stats.disk_usage_total,
+    }
+
+
+def collate_field_usage_stats(stats):
```

Comment:
```suggestion
def collate_disk_usage_stats(stats):
```

Reply:
I only suggested this one fix, but you need to fix the callers too, sorry.

Reply:
That's fair. I'll fix.